mne/commands/tests/test_commands.py

```diff
@@ -177,4 +178,28 @@ def test_surf2bem():
     check_usage(mne_surf2bem)
 
 
+@slow_test
+@requires_freesurfer
```

Comment:
I agree on the proposition and the remarks from Eric.

Let me know when I shall review.